gateway_to_backend/README.md

```diff
@@ -277,6 +277,24 @@ _Even if this version is increased, the API remains backward compatible.
 It just help backends development to identify if new features are
 present in a gateway._
 
+#### Get gateway status
+
+- **Request:**
+
+    > **topic:** gw-request/get_gw_status
```

Comment:
Should this have `<gw-id>`?

Reply:
Seems to trigger for all gateways.

Reply:
Yes, exactly, the goal of this new message was to request the status of all gateways to know which ones are online. Therefore, gw-id is not present in the topic.